pennylane/devices/default_tensor.py

```diff
@@ -352,8 +352,7 @@ def circuit(phi, depth, num_qubits):
             The execution time for this circuit with the above parameters is around 0.8 seconds on a standard laptop.
 
             The tensor network method can be faster than MPS and state vector methods in some cases.
-            As a comparison, the time for the exact calculation of the same circuit with the MPS method and with the ``default.qubit``
-            device is about three orders of magnitude slower.
+            As a comparison, the time for the exact calculation (i.e., with ``max_bond_dim = None``) of the same circuit with the `default.tensor` device using the MPS method is approximately three orders of magnitude slower. Using the ``default.qubit`` device also results in a much slower simulation.
```

Comment:
I forgot the double ` in my suggestion 

```suggestion
            As a comparison, the time for the exact calculation (i.e., with ``max_bond_dim = None``) of the same circuit with the ``default.tensor`` device using the MPS method is approximately three orders of magnitude slower. Using the ``default.qubit`` device also results in a much slower simulation.
```

Reply:
Also, it would be ideal to change the line to make the source code more readable. Right now, this is what it looks like:

![line_change](https://github.com/PennyLaneAI/pennylane/assets/68463840/b5cf6c87-88f4-4f7f-a4e4-dd106a70a1e8)

I don't know how to change the line in the code suggestion on GitHub without risking messing up the version in the doc. I can push a small change myself or you can do it yourself, whatever you prefer. Thanks!

Reply:
@EmilianoG-byte Looks great now!

Reply:
@EmilianoG-byte Notice that MPS does not have the ` above in the text, That is, it's always written as MPS, not ``MPS``